src/disk_interface_test.cc

```diff
@@ -42,6 +43,15 @@ struct DiskInterfaceTest : public testing::Test {
     return fclose(f) == 0;
   }
 
+#ifdef _WIN32
+  bool Touch(const wchar_t* path) {
```

Comment:
There's not a more direct way to change the modification time of a file on Windows? That seems hard to believe?